### PR TITLE
Feature/oui stepper

### PR DESCRIPTION
**Pull request by AxelPeter**
UU-742
http://build_oui_stepper.ui-kit.ovh

Linked to:
https://github.com/ovh-ux/ovh-ui-angular/pull/149
https://github.com/ovh-ux/ovh-ui-kit-documentation/pull/87

### CODE REVIEW

File: packages/oui-list/README.md [hunk 1]
```diff
@@ -138,94 +138,3 @@
   </ul>
 </div>
 ```
-
-### Steps
```
Comment (tremblaymath) [inline packages/oui-list/README.md]: 👍

File: packages/oui-list/_mixins.less [hunk 2]
```diff
@@ -188,203 +97,4 @@
...
-    .@{selector}__item {
-      border-bottom: @separator-size solid @separator-color;
-
-      &:last-child {
-        border-bottom: 0 none;
-      }
-    }
-  }
-
-  .list-steps (
```
Comment (tremblaymath) [inline packages/oui-list/_mixins.less]: It doesn't cause breaking changes? only a move to packages/oui-list/list.less?

File: packages/oui-list/_variables.less [hunk 3]
```diff
@@ -35,9 +35,10 @@
 
 // List - Steps
 @oui-list-steps-font-color: #0b4571;
-@oui-list-steps-icon-color: #d5eef5;
-@oui-list-steps-disabled-font-color: fade(@oui-list-steps-font-color, 30%);
-@oui-list-steps-disabled-icon-color: #fff;
+@oui-list-steps-icon-font-color: #fff;
+@oui-list-steps-icon-color: #113F6D;
```
Comment (tremblaymath) [inline packages/oui-list/_variables.less]: each color should have a name and be centralized for all component

File: packages/oui-stepper/_variables.less [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+@oui-stepper-step-icon-color_disabled: @oui-color-dusty;
+@oui-stepper-step-icon-font-size: rem-calc(20);
+@oui-stepper-step-icon-font-weight: @oui-font-semibold;
+@oui-stepper-step-icon-size: rem-calc(30);
+@oui-stepper-step-icon-top: rem-calc(5);
+
+@oui-stepper-step-title-line-height: rem-calc(22);
+@oui-stepper-step-title-color: @oui-color-sapphire;
+@oui-stepper-step-title-color_complete: @oui-color-sapphire;
+@oui-stepper-step-title-color_disabled: #b3b3b3;
```
Comment (tremblaymath) [inline packages/oui-stepper/_variables.less]: same "each color should have a name and be centralized for all component"

File: packages/oui-list/_mixins.less (hunk 2 above)
Comment (AxelPeter) [reply packages/oui-list/_mixins.less]: Yes, I just move the useless mixins in the main file :)

File: packages/oui-stepper/_variables.less (hunk 4 above)
Comment (AxelPeter) [reply packages/oui-stepper/_variables.less]: It's use this way in others components. 
Maybe we should use `@oui-color-silver` instead (`#bbbdbf`)

**Comment by AxelPeter** [comment]
I've done a rebase for conflicts

**Pull request merged**